app/keepers/types.go

```diff
@@ -0,0 +1,80 @@
+package keepers
+
+import (
```

Comment:
Per comment #1 in https://github.com/pokt-network/poktroll/pull/702#discussion_r1697594114. Can you please:

1. Explain why we have diverged from cosmos standards
2. Link to this PR in the comment for reference
3. Mention ignite might stop working.

Reply:
1. Done. 
Sure! To be fair, ignite scaffolding already doesn't work all the way and requires just a little of manual adjustment due to how our protos/packages are located.